index.php

```diff
@@ -7,23 +7,86 @@
 $dbName = 'collectorapp';
 
 $PDO = connectDb($host, $username, $password, $dbName);
-$bottles = getBottles($PDO);
 
+if(count($_POST)>0){
+    addBottle($PDO, $_POST['itemname'], $_POST['purchaselocation'], $_POST['type'], $_POST['purchasedate']);
+}
 
-?>
-<html lang="en">
+$bottles = getBottles($PDO);
 
+?>
+<!DOCTYPE html>
+<html lang="en-GB">
 <head>
     <title>Collector App</title>
+    <link rel="stylesheet" href="normalize.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;1,400&display=swap" rel="stylesheet">
+    <link rel="shortcut icon" type="image/jpg" href="favicon.ico"/>
 </head>
-
 <body>
-
+    <header>
+        <h1>Collector App</h1>
+    </header>
     <main>
+        <section class="new-item-form-section">
+            <form method="POST" action="index.php">
+                <h2>Add a new bottle to your collection</h2>
+                <div class="form-outer">
+                    <div class="form-inner">
+                        <div>
+                            <label for="itemname">Bottle name</label>
+                            <input type="text" id="itemname" name="itemname" required>
+                        </div>
+                        <div>
+                            <label for="type">Type of alcohol</label>
+                            <select id="type" name="type" required>
+                                <option value="rum">Rum</option>
+                                <option value="rye">Rye</option>
+                                <option value="vodka">Vodka</option>
+                                <option value="whisky">Whisky</option>
+                            </select>
+                        </div>
+                    </div>
+                    <div class="form-inner">
+                        <div>
+                            <label for="purchaselocation">Purchase location</label>
+                            <input type="text" id="purchaselocation" name="purchaselocation" required>
```

Comment:
this id is all lowercase whereas some other classes within the document are kebab-case. it would be better if everything is consistently named

Reply:
including the `name` attribute as well